server/containers/genrepo/process_incoming.py

```diff
@@ -147,6 +147,9 @@ def make_index(bucket: s3.Bucket, prefix: pathlib.Path, pkg_dir: str) -> None:
 
         basename = m.group("basename")
         slot = m.group("slot") or ""
+        installref = obj.key
+        if not installref.startswith('/'):
```

Comment:
Can we just do this?
```
ARCHIVE = pathlib.Path("/archive")
```

Reply:
That would require rewriting existing S3 objects, and AFAIK leading slash in the object name is breaking the bucket UI or something like that.